os/binfmt/binfmt_loadbinary.c

```diff
@@ -47,6 +47,8 @@
 #include <tinyara/mm/mm.h>
 
 #ifdef CONFIG_SUPPORT_COMMON_BINARY
+#include <tinyara/userspace.h>
+struct userspace_s *USERSPACE;
```

Comment:
It looks strange to define this here.
I think we should define this in upspace module and should take it as extern.

Reply:
Even up_userspace funtion is not used, it will be compiled because you don't block it at Makefile. Then, we could use that file to define userspace I think.

Reply:
Ok. Done.